.github/workflows/diff_report.yml

```diff
@@ -134,8 +136,38 @@ jobs:
        id: out
        run: echo ::set-output name=message::$(cat message)   
       
-     - name: Comment PR
-       uses: ./contribution/comment-action/
-       with:
-        message: ${{steps.out.outputs.message}}
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  # should be always last step
+  send_message:
+    runs-on: ubuntu-latest
+    needs: [parse_body, make_report]
+    if: failure() || success()
+    steps:
+
+      - name: Success message
+        if: needs.make_report.outputs.message != ''
+        run: echo "${{needs.make_report.outputs.message}}" > message
+      
+      - name: Failure message
+        if: needs.make_report.outputs.message == ''
```

Comment:
in line 143 it is 
`if: failure() || success()`
is it possible to use same function here?